content/3_Silver/Graph_Traversal.mdx

```diff
@@ -380,7 +405,13 @@ public static void main(String[] args) throws IOException {
 	ArrayList<ArrayList<Integer>> adj = new ArrayList<ArrayList<Integer>>();
 	for (int i = 0; i < MAX_N; i++) adj.add(new ArrayList<Integer>());
 
-	// Define adjacency list and read in problem-specific input
+	/*
+	 * Define adjacency list and read in problem-specific input
+	 *
+	 * In this example, we've provided "dummy input" that's
+	 * reflected in the GIF above to help illustrate the
+	 * order of the recrusive calls.
+	 */
 
 	for (int i = 0; i < MAX_N; i++) {
```

Comment:
@SansPapyrus683

if this is okay with you, could you also make the changes to the java code?